pages/common/gh-screensaver.md

```diff
@@ -0,0 +1,25 @@
+# gh screensaver
+
+> Extension for GitHub CLI that runs animated terminal screensavers.
+> See also: `gh extension`.
+> More information: <https://github.com/vilmibm/gh-screensaver>.
+
+- Run a random screensaver:
```

Comment:
What do you think about this?
```suggestion
- Start a random screensaver:
```

Reply:
Not sure, the screensaver runs inside the terminal until any keypress.

Reply:
How about this?
```suggestion
- Start a random screensaver in the terminal:
```
Otherwise, the original is okay too.

Reply:
`Start` sounds worse to me honestly

Reply:
Either start or run would work fine here.